docs/index.rst

```diff
@@ -3,7 +3,7 @@ Solidity
 
 Solidity is a high-level language whose syntax is similar to that of JavaScript
 and it is designed to compile to code for the Ethereum Virtual Machine.
-As you will see, it is quite easy to create contracts for voting,
+As you will see, it is possible to create contracts for voting,
```

Comment:
I would still say that it is easy. We could perhaps add some warning that there are malicious actors out there and they can steal your money if you do not get it perfectly right and that people should think about how they use their smart contracts.

I mean I can use crappy php code (nothing specifically against php here :-) ) on a webserver that stores my grocery list. I would take a closer look at the code if it would manage my bank account.